bouter/free/__init__.py

```diff
@@ -1,6 +1,149 @@
 from bouter import Experiment
+from bouter import utilities, get_scale_mm
+import numpy as np
 
 
 class FreelySwimmingExperiment(Experiment):
-    def bouts(self):
-        pass
+
+    def get_n_segments(self, prefix=True):
```

Comment:
this should be actually extracted from the tracking metadata now, look at how n_segments are in the embedded experiments now (analogously for n_fish)